sound-bubble/imports/ui/components/Home/Vote.jsx

```diff
@@ -25,7 +25,9 @@ class Vote extends React.Component {
               <span className="tooltiptext">Undo Upvote</span>
             </div>
           </div>
```

Comment:
Ah, I restyled the vote behaviour in my styling branch as well. I guess we can pick which one we want.

Reply:
alright, I just using different colour to distinguish between upvote amount and downvote amount

Reply:
I think Martin had the colours too lol. One vote for keeping Martin's change.